src/Support/SupportData.php

```diff
@@ -551,18 +557,33 @@ static function ( $plugin ) {
 		$allowed_types  = [ 'plugin', 'theme' ];
 		$source['type'] = ( ! empty( $source['type'] ) ) ? strtolower( trim( $source['type'] ) ) : '';
 
+		if ( ! empty( $source['sources'] ) && is_array( $source['sources'] ) ) {
+			foreach ( $source['sources'] as $index => $inner_source ) {
+				$source['sources'][ $index ] = self::normalize_error_source( $inner_source );
+			}
+
+			$source['sources'] = array_values( array_filter( $source['sources'] ) );
+		}
+
+
 		/**
 		 * Do not include wp-core sources.
+		 * But allow if source have sub sources.
 		 */
-		if ( empty( $source['type'] ) || ! in_array( $source['type'], $allowed_types, true ) ) {
+		if (
+			( empty( $source['type'] ) || ! in_array( $source['type'], $allowed_types, true ) ) &&
```

Comment:
@dhaval-parekh Maybe move the `&&` into new line to improve readability:

```suggestion
			( empty( $source['type'] ) || ! in_array( $source['type'], $allowed_types, true ) )
			&&
			...
```